gammapy/background/tests/test_fov.py

```diff
@@ -66,13 +66,13 @@ def test_fill_acceptance_image():
     xpix_coord_grid, ypix_coord_grid = coordinates(image, world=False)
 
     # calculate pixel offset from center (in world coordinates)
-    coord = pixel_to_skycoord(xpix_coord_grid, ypix_coord_grid, w, 0)
+    coord = pixel_to_skycoord(xpix_coord_grid, ypix_coord_grid, w, origin=0)
     pix_off = coord.separation(center)
 
-    # x axis defined in the array positions [y_center_pix - 1,:]
-    # only interested in semi axis, so [y_center_pix - 1, x_center_pix - 1:]
-    ix_min = int(round(x_center_pix - 1))
-    iy = int(round(y_center_pix - 1))
+    # x axis defined in the array positions [y_center_pix,:]
+    # only interested in semi axis, so [y_center_pix, x_center_pix:]
+    ix_min = int(round(x_center_pix))
```

Comment:
This fails on Python 3:
https://travis-ci.org/gammapy/gammapy/jobs/57998355#L1045